config/settings.yml

```diff
@@ -30,7 +30,7 @@ feature_flags:
 traject:
   processing_thread_pool: 1
 iiif_embed:
-  url: https://embed.stanford.edu/iiif
+  url: https://embed-stage.stanford.edu/iiif
```

Comment:
```suggestion
  url: https://embed-stage.stanford.edu/iiif
  url: http://localhost:3001.stanford.edu/iiif
```
For testing locally

Reply:
The place for local values is  `config/settings/developement.yml`